tools/oracle2bq-schema-converter/example_terraform_dir/terraform copy.tfvars

```diff
@@ -0,0 +1,94 @@
+project_id                  = "project_id"
```

Comment:
seems like this does not need to be checked into VCS as we have another example.